samples/nrf5340/multiprotocol_rpmsg/README.rst

```diff
@@ -0,0 +1,70 @@
+.. _multiprotocol-rpmsg-sample:
+
+Multiprotocol RPMsg
+###################
+
+Overview
+********
+
+This sample exposes :ref:`softdevice_controller` and IEEE 802.15.4 radio driver support to the nRF5340 application CPU using RPMsg transport which is a part of `OpenAMP <https://github.com/OpenAMP/open-amp/>`__.
+
+The sample requires a nRF5340 development kit. The
+firmware must be flashed to the nRF5340 network core.
+This sample is compatible with the HCI RPMsg driver provided by Zephyr's Bluetooth :ref:`bt_hci_drivers` core.
```

Comment:
Should we state compatibility with 802.15.4 counterpart as well? Something like:
...and nRF 802.15.4 Serialization Host module